packages/vscode-mdx/src/typescript-plugin.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @typedef {import('typescript/lib/tsserverlibrary.js').server.Project} Project
+ * @typedef {import('typescript/lib/tsserverlibrary.js').server.PluginModuleFactory} PluginModuleFactory
+ * @typedef {import('typescript/lib/tsserverlibrary.js').LanguageService} LanguageService
```

Comment:
This is equivalent nowadays.

```suggestion
 * @typedef {import('typescript').server.Project} Project
 * @typedef {import('typescript').server.PluginModuleFactory} PluginModuleFactory
 * @typedef {import('typescript').LanguageService} LanguageService
```